flake.nix

```
@@ -74,8 +74,20 @@
         ${getExe pkgs.reuse} --suppress-deprecation lint
         ${getExe pkgs.lychee} --offline --no-progress .
         ${getExe lint-python}
+        ${getExe format-cpp}
       '';
 
+      format-cpp = pkgs.writeShellApplication {
+        name = "format-cpp";
+        runtimeInputs = [pkgs.clang-tools_18];
```

We should really be using the clang-format from the CHERIoT toolchain.

```suggestion
        runtimeInputs = with lrPkgs; [llvm_cheriot];
```

---

The clang-format from CHERIot toolchain seems to be using an old version that does not allow configuring the `AllowShortLoopsOnASingleLine`.
See details: https://github.com/llvm/llvm-project/issues/61708